File: Mobile/app/src/main/java/com/example/studentmobileapp/Models/Workshop.kt
package com.example.studentmobileapp.Models

import kotlinx.serialization.Serializable
import java.util.Date

@Serializable
class Workshop {
  var workshopId: Int = 0
  var title: String = ""
  var category: Category? = null
  var status: String = ""
  var requestDate: String = ""
  var timeslot: String = ""
  var endDate: String = ""
  var startDate: String = ""
  var saloon: Saloon? = null
}

File: Desktop/Desktop/Models/Workshop.cs
using System;
using System.Collections.Generic;

// Code scaffolded by EF Core assumes nullable reference types (NRTs) are not used or disabled.
// If you have enabled NRTs for your project, then un-comment the following line:
// #nullable disable

namespace Desktop.Models
{
    public partial class Workshop
    {
        public int WorkshopId { get; set; }
        public int UserId { get; set; }
        public string Title { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public int Timeslot { get; set; }
        public int SaloonId { get; set; }
        public int CategoryId { get; set; }
        public int Status { get; set; }
        public DateTime RequestDate { get; set; }

        public virtual Category Category { get; set; }
        public virtual Saloon Saloon { get; set; }
        public virtual User User { get; set; }
    }
}
